src/test/java/ru/training/at/hw2/ex1/ExerciseFirst.java

```diff
@@ -0,0 +1,177 @@
+package ru.training.at.hw2.ex1;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import ru.training.at.hw2.TestBaseClass;
+
+public class ExerciseFirst extends TestBaseClass {
+
+    SoftAssert softAssert = new SoftAssert();
+
+    @BeforeTest
+    @Override
+    public void beforeTest() {
+        super.beforeTest();
+    }
+
+    @Test
+    public void homePageTestWithSoftAsserts() {
+
+        //1.Open site
+        webDriver.get("https://jdi-testing.github.io/jdi-light/index.html ");
+
+        //2.Assert Browser title
+        (new WebDriverWait(webDriver, 10))
+                .until(ExpectedConditions.titleContains(webDriver.getTitle()));
+        String actual = webDriver.getTitle();
+        String expected = "Home Page";
+        softAssert.assertEquals(actual, expected);
+
+        //3.Perform login
+        WebElement loginButton = (new WebDriverWait(webDriver, 10)
+                .until(ExpectedConditions.elementToBeClickable(webDriver.findElement(By.id("user-icon")))));
+        loginButton.click();
+        WebElement username = (new WebDriverWait(webDriver, 10)
+                .until(ExpectedConditions.presenceOfElementLocated(By.id("name"))));
+        username.sendKeys("Roman");
+        WebElement password = (new WebDriverWait(webDriver, 10)
+                .until(ExpectedConditions.presenceOfElementLocated(By.id("password"))));
+        password.sendKeys("Jdi1234");
+        WebElement enterButton = (new WebDriverWait(webDriver, 10)
+                .until(ExpectedConditions.presenceOfElementLocated(By.xpath("//*[@id='login-button']"))));
+        enterButton.click();
```

Comment:
May be it make sense create one variable for the `WebDriverWait` and do not create new instance of it every time?